src/server/kernel/wtoplevelsurface.h

```diff
@@ -50,6 +50,13 @@ class WToplevelSurface : public QObject
         return QSize();
     }
 
+    virtual int keyboardFocusPriority() const {
```

Comment:
Use enum?

Reply:
This is not `keyboard_interactivity`, it's a comprehensive consideration of the value of `keyboard_interactivity` and `layer`. not need to return enum.